### ISSUE TITLE
Bug 1226275 - mgradm upgrade podman should not run upgrade is version is the same 

### ISSUE THREAD

**Issue by mbussolotto** [uyuni-project/uyuni-tools#361]
Currently mgradm upgrade podman run steps required for upgrade also if the new version == current version. Although it might be useful for devs, it might be misleading for users. Better to perform some check and add a --force.

https://bugzilla.suse.com/show_bug.cgi?id=1226275

**Comment by rjmateus** [comment]
What will be the image version in case of a PTF? Should (or can) this check be made on the PTF install also?

**Comment by mbussolotto** [comment]
> What will be the image version in case of a PTF? Should (or can) this check be made on the PTF install also?

good point, in case of `support ptf` the ugprade should be performed also if version isn't changed

**Comment by vzhestkov** [comment]
Closing in favor of https://github.com/SUSE/spacewalk/issues/24562
